homeassistant/components/isy994/climate.py

```diff
@@ -0,0 +1,260 @@
+"""Support for Insteon Thermostats via ISY994 Platform."""
+from typing import Callable, List, Optional, Union
+
+from pyisy.constants import (
+    CMD_CLIMATE_FAN_SETTING,
+    CMD_CLIMATE_MODE,
+    ISY_VALUE_UNKNOWN,
+    PROP_HEAT_COOL_STATE,
+    PROP_HUMIDITY,
+    PROP_SETPOINT_COOL,
+    PROP_SETPOINT_HEAT,
+    PROP_UOM,
+    PROTO_INSTEON,
+)
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    DOMAIN as CLIMATE,
+    FAN_AUTO,
+    FAN_ON,
+    HVAC_MODE_COOL,
+    HVAC_MODE_HEAT,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+    SUPPORT_TARGET_TEMPERATURE_RANGE,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    PRECISION_TENTHS,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+)
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import (
+    _LOGGER,
+    DOMAIN as ISY994_DOMAIN,
+    HA_FAN_TO_ISY,
+    HA_HVAC_TO_ISY,
+    ISY994_NODES,
+    ISY_HVAC_MODES,
+    UOM_DOUBLE_TEMP,
+    UOM_FAN_MODES,
+    UOM_HVAC_ACTIONS,
+    UOM_HVAC_MODE_GENERIC,
+    UOM_HVAC_MODE_INSTEON,
+    UOM_ISY_CELSIUS,
+    UOM_ISY_FAHRENHEIT,
+    UOM_ISYV4_DEGREES,
+    UOM_ISYV4_NONE,
+    UOM_TO_STATES,
+)
+from .entity import ISYNodeEntity
+from .helpers import migrate_old_unique_ids
+
+ISY_SUPPORTED_FEATURES = (
+    SUPPORT_FAN_MODE | SUPPORT_TARGET_TEMPERATURE | SUPPORT_TARGET_TEMPERATURE_RANGE
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType,
+    entry: ConfigEntry,
+    async_add_entities: Callable[[list], None],
+) -> bool:
+    """Set up the ISY994 thermostat platform."""
+    devices = []
```

Comment:
```suggestion
    entities = []
```

Reply:
Do you want me to pick this up for the other platforms too?

Reply:
Yes, this is the standard.